selfdrive/car/tests/routes.py

```diff
@@ -129,6 +129,7 @@
   CarTestRoute("007d5e4ad9f86d13|2021-09-30--15-09-23", HYUNDAI.KIA_K5_2021),
   CarTestRoute("c58dfc9fc16590e0|2023-01-14--13-51-48", HYUNDAI.KIA_K5_HEV_2020),
   CarTestRoute("50c6c9b85fd1ff03|2020-10-26--17-56-06", HYUNDAI.KIA_NIRO_EV),
+  CarTestRoute("b153671049a867b3|2023-03-31--13-16-13", HYUNDAI.KIA_NIRO_EV_2ND_GEN),
```

Comment:
It looks like the logs for this route are gone, can you provide a new one with rlogs uploaded? (you can click the upload button for "All logs" in connect)

Reply:
Can you try again ? I just clicked on the upload button for "All logs" on this route.

Reply:
They're still missing. It looks you may have a smaller sized C3 variant, which won't store as many logs. I would try to get a route then quickly upload to WiFi after

Reply:
Ok, check this one instead : b153671049a867b3|2023-04-05--10-00-30

Reply:
That works, thanks!